### PR TITLE
[v3.3] Bump to Buildah v1.22.0 [NO NEW TESTS NEEDED]

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
[NO TESTS NEEDED]

Bump Buildah to v1.22.0 and c/common to v0.42.1
and adds a patch for cpu handling.

Based on #1132

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
Repushed without the word "NEW" in the test tag, hopefully that will work.

**Comment by TomSweeneyRedHat** [comment]
@containers/podman-maintainers can I get another "LGTM" and a merge please?
All happy green test buttons.

**Review by vrothberg — COMMENTED** [review]
/lgtm

**Review by saschagrunert — APPROVED** [review]

**Pull request merged**
